src/basics.php

```diff
@@ -51,7 +51,10 @@ function debug($var, $showHtml = null, $showFrom = true)
         $line = '';
         $lineInfo = '';
         if ($showFrom) {
-            $trace = Debugger::trace(['start' => 1, 'depth' => 2, 'format' => 'array']);
+            $trace = Debugger::trace(['start' => 1, 'depth' => 3, 'format' => 'array']);
+            if (count($trace) > 1) {
```

Comment:
Could walk the stack and find the call containing debug, dd, or other functions and take the next frame?